vm/interpreter/src/default_runtime.rs

```diff
@@ -611,10 +615,15 @@ where
         rand_epoch: ChainEpoch,
         entropy: &[u8],
     ) -> Result<Randomness, ActorError> {
-        let r = self
-            .rand
-            .get_chain_randomness(personalization, rand_epoch, entropy)
-            .map_err(|e| e.downcast_fatal("could not get randomness"))?;
+        let r = if rand_epoch > networks::UPGRADE_PLACEHOLDER_HEIGHT {
```

Comment:
The placeholder is currently set to `9_999_999`, when will this be set?

Reply:
Should have it set once we update the actors interface.